File: src/Math/NumberOfSteps/python/NumberOfSteps_test.py
import pytest
from NumberOfSteps.python.numberOfSteps import Solution
from NumberOfSteps.python.testCases import runTestCase, cases


@pytest.mark.parametrize('case', cases)
def test(case):
    solution = Solution()
    assert runTestCase(solution.numberOfSteps, case) == case['expected']

File: src/Math/Easy/NumberOfSteps/javascript/numberOfSteps.test.js
import solution from './numberOfSteps.es3.cjs';
import runTestCase, { cases } from './testCases.js';

test.each(cases.map(c => [c.name, c]))(
  '[Math/Easy/[1342] Number of Steps to Reduce a Number to Zero] - Case %#',
  (name, testCase) => expect(runTestCase(solution, testCase)).toEqual(testCase.expected),
);
